triplea-game/triplea
# Use Interface Types for ArrayList, HashSet and HashMap

## Overview
This changes the declaration type of all ArrayList, HashSet and HashMap declarations to their respective interface type.

## Functional Changes
In theory none, I'm not entirely sure if a changed method signature could potentially break network compatibility to the lobby, I'll check this evening because I'm currently a bit in a hurry.

## Manual Testing Performed
None, yet.
EDIT: I verified I could join the lobby no problem, and could connect to the bot, (which told be that my version is too high) after disabling the client side version check,
